specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,509 @@
+    winrt::hstring group = L"downloads";
+
+    // Assign new values
+    winrt::AppNotificationProgressData data(2 /* Sequence number */ );
+    data.Title(L"Weekly playlist"); // Binds to {progressTitle} in xml payload
+    data.Value(0.7); // Binds to {progressValue} in xml payload
+    data.ValueStringOverride(L"18/26 songs"); // Binds to {progressValueString} in xml payload
+    data.Status(L"Downloading..."); // Binds to {progressStatus} in xml payload
+
+    auto result = co_await winrt::AppNotificationManager::Default().UpdateAsync(data, tag, group);
+    if (result == winrt::AppNotificationProgressResult::AppNotificationNotFound)
+    {
+        LOG_HR_MSG(E_NOT_SET, "Toast Progress Update Failed since the previous notification update is missing or has been cleared by the user!");
+    }
+}
+```
+
+# Remarks
+
+## Registration
+
+The developer should always call the Notification Registration API first to register the current
+process as the target for Notification invokes. When the app is terminating, the developer should
+call Unregister() to free up COM and allow for subsequent invokes to launch a new process.
+
+## Event Handlers
+
+The developer should subscribe to Notification event handlers to handle invocations if the process
+happens to be already running. If this isn't done, the OS will launch a new process for every
+subsequent invocation.
+
+## Manifest Registration
+
+For MSIX, the COM activator GUID and the exe need to be specified in the manifest as part of the
+desktop and com extensions. In the example below, we use an activatorId =
+"1920DBA9-0F64-4F0D-8A4B-5E207B812E61". The launch args would need to be pre-set to
+"----AppNotificationActivated:" to ensure that WinAppSDK can process the payload as an
+AppNotification Kind.
+
+```xml
+<Extensions>
+    <desktop:Extension Category="windows.toastNotificationActivation">
+		<desktop:ToastNotificationActivation ToastActivatorCLSID="1920DBA9-0F64-4F0D-8A4B-5E207B812E61" />
+	</desktop:Extension>
+    <com:Extension Category="windows.comServer">
+        <com:ComServer>
+            <com:ExeServer Executable="SampleBackgroundApp\SampleBackgroundApp.exe" DisplayName="SampleBackgroundApp" Arguments="----AppNotificationActivated:">
+                <com:Class Id="1920DBA9-0F64-4F0D-8A4B-5E207B812E61" />
+            </com:ExeServer>
+        </com:ComServer>
+    </com:Extension>
+</Extensions>
+```
+
+# API Details
+
+```c#
+namespace Microsoft.Windows.AppNotifications
+{
+    // Event args for the Notification Activation
+    [experimental]
+    runtimeclass AppNotificationActivatedEventArgs
+    {
+        // Arguments from the invoked button. Empty for Default Activation with no launch args specified in payload.
+        String ActivationArgs{ get; };
+
+        // The data from the input elements of a Notification like a TextBox
+        Windows.Foundation.Collections.IMap<String, String> UserInput{ get; };
+    }
+
+    // Notification Progress Data
+    [experimental]
```

Comment:
these `[experimental]` attributes need to go.

Reply:
Removed all experimental attributes. We will do this in our PR as well for Public Preview